execution/executor/src/lib.rs

```diff
@@ -743,7 +743,11 @@ impl<V: VMExecutor> BlockExecutor for Executor<V> {
         let version = ledger_info_with_sigs.ledger_info().version();
         let num_persistent_txns = self.cache.synced_trees().txn_accumulator().num_leaves();
 
-        if version + 1 < num_persistent_txns {
+        if version
```

Comment:
nit: part of me wonders if it'll be cleaner to create a helper for this (i.e., to avoid having to create a new format error with "version + 1 overflows", twice)?

Reply:
that's what I did here: https://github.com/libra/libra/pull/4991/files#diff-f557c4f487f2988c1ebb753153774837R108